modules/querying/pages/output-statements-and-file-objects.adoc

```diff
@@ -3,15 +3,18 @@
 [#_print_statement_api_v2]
 == `PRINT` Statement (API v2)
 
-The PRINT statement specifies output data.Each execution of a PRINT statement adds a JSON object to the results array which will be part of the query output.A PRINT statement can appear anywhere that query-body statements are permitted.
+The `PRINT` statement specifies output data.
+Each execution of a `PRINT` statement adds a JSON object to the results array which will be part of the query output.
+A `PRINT` statement can appear anywhere that query-body statements are permitted.
```

Comment:
Is "anywhere that" grammatical? I think "that" must be a subject or object in the clause. 

A `PRINT` statement can appear wherever query-body statements are permitted.